app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -68,6 +68,18 @@ def datastores(opts = {})
       end
     end
 
+    def storage_pods(opts = {})
+      if opts[:storage_pod]
+        dc.storage_pods.get(opts[:storage_pod]) rescue {}
```

Comment:
Mind to rescue the appropriate possible Fog / network exceptions? This can hide errors about the connection between Foreman and vSphere that can be hard to debug.